src/utils/image.js

```diff
@@ -33,6 +33,8 @@ export function getThumbnailType(file) {
 export function useFileData(initialFiles, onSetFiles = () => {}) {
   const [files, setFiles] = useState(toList(initialFiles));
   useEffect(() => {
+    // prevent infinite loop of setting a new empty array as the value
+    if (files.length === 0 && initialFiles.length === 0) return;
```

Comment:
why does this infinite loop? doesn't look like any of these actions should affect initialFiles

Reply:
I changed initialFiles default value to be `[]` from `null` and that triggers an infinite loop with the useEffect unless I add this conditional

Reply:
make this static